index.d.ts

```diff
@@ -5,6 +5,12 @@ export interface Options {
 	 * @default Infinity
 	 */
 	concurrency?: number;
+	/**
+	 * Throw an aggregate error at the end when all promise are settled.
```

Comment:
This description makes it sound like it will throw an error no matter what.

The description needs to be better. It should also explain what an aggregate error is and link to the package.

You also need to document this in the readme too.